example/WOWserify/README.md

```diff
@@ -0,0 +1,9 @@
+# WOWserify
+
+A sample page displaying dogescript on one text area, and javascript in another.
```

Comment:
I opened a pull request to this branch here https://github.com/AnEmortalKid/dogescript/pull/3; I'd love to remove all of this generated code from git if possible